update-script.js

```diff
@@ -0,0 +1,141 @@
+const KintoClient = require("kinto-http").default;
+const btoa = require("btoa");
+const fetch = require("node-fetch");
+const AppConstants = require("./app-constants");
+
+const WEBSITES_WITH_SHARED_CREDENTIAL_COLLECTION = "websites-with-shared-credential-backends";
+/** @type {String} */
+const FX_RS_WRITER_USER = AppConstants.FX_REMOTE_SETTINGS_WRITER_USER;
+/** @type {String} */
+const FX_RS_WRITER_PASS = AppConstants.FX_REMOTE_SETTINGS_WRITER_PASS;
+/** @type {String} */
+const SERVER_ADDRESS = AppConstants.FX_REMOTE_SETTINGS_WRITER_SERVER;
+
+const APPLE_API_ENDPOINT = "https://api.github.com/repos/apple/password-manager-resources/contents/quirks/websites-with-shared-credential-backends.json";
+
+/**
+ * Fetches the source records from the APPLE_API_ENDPOINT
+ *
+ * @return {String[][]} 
+ */
+const getSourceRecords = async () => {
+  const response = await fetch(APPLE_API_ENDPOINT, {
+    headers: {
+      "Accept": "application/vnd.github.v3.raw"
```

Comment:
There's no need for Github token? Are you sure you won't be annoyed by rate limiting?

Reply:
According to [Github's API docs](https://docs.github.com/en/rest/overview/resources-in-the-rest-api), unauthenticated requests are rate limited at 60 requests per hour. Since this script should only run once every two weeks (or so), I don't foresee any issue with rate limiting.

Is there something I'm missing here?

Reply:
I don't think the rate limiting is a problem. An alternative completely sidestepping the question would be to get the contents from https://raw.githubusercontent.com/apple/password-manager-resources/main/quirks/websites-with-shared-credential-backends.json (albeit with a content type of `text/plain`).